### PR TITLE
Add release notes for 7.10.1

### PR DESCRIPTION
**Pull request by fcofdez**
https://elasticsearch_65439.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/7.10/release-notes-7.10.1.html

### CODE REVIEW

File: docs/reference/release-notes/7.10.asciidoc
Comment (jrodewig) [inline docs/reference/release-notes/7.10.asciidoc:54]: This label is redundant.
```suggestion
* Fix the return type problem in the sign function {es-pull}64845[#64845]
```

File: docs/reference/release-notes/7.10.asciidoc
Comment (jrodewig) [inline docs/reference/release-notes/7.10.asciidoc:47]: @lcawl @szabosteve Y'all mind taking a look at these?

**Review by jrodewig — APPROVED** [review]
LGTM other than a minor nit. I'll defer to @lcawl and @szabosteve on the ML section. Thanks @fcofdez!

**Review by szabosteve — APPROVED** [review]
LGTM, thanks @fcofdez! I've added one more ML related item to the list.

**Comment by chriscressman** [comment]
CI for https://github.com/elastic/docs/pull/2012 won't pass until this merges. No rush, but if there are no further changes expected, can we merge this? Thanks!

**Comment by fcofdez** [comment]
Sorry, I wasn't sure when this should be merged.

**Comment by lcawl** [comment]
FYI I added a "coming" tag in 5320083bedb8acb1f026c1d913135ad336cc4726 so it doesn't look like the release is already out.

**Comment by chriscressman** [comment]
@fcofdez No worries at all. Thanks everyone :smiling_face_with_three_hearts:

**Pull request merged**
